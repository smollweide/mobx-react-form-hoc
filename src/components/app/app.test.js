import React from "react";
import ReactDOM from "react-dom";
import { shallow } from "enzyme";
import App from "./app";

it("renders without crashing", () => {
    const div = document.createElement("div");
    ReactDOM.render(<App />, div);
});

it("renders correctly according to the snapshot", () => {
    expect(shallow(<App />)).toMatchSnapshot();
});
