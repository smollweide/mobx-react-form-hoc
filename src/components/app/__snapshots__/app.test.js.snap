exports[`test renders correctly according to the snapshot 1`] = `
ShallowWrapper {
  "complexSelector": ComplexSelector {
    "buildPredicate": [Function],
    "childrenOfNode": [Function],
    "findWhereUnwrapped": [Function],
  },
  "length": 1,
  "node": <div>
    <h2>
        mobx-react-form-hoc demo
    </h2>
    <withForm(LoginForm) />
</div>,
  "nodes": Array [
    <div>
      <h2>
            mobx-react-form-hoc demo
      </h2>
      <withForm(LoginForm) />
</div>,
  ],
  "options": Object {},
  "renderer": ReactShallowRenderer {
    "_instance": ShallowComponentWrapper {
      "_calledComponentWillUnmount": false,
      "_compositeType": 2,
      "_context": Object {},
      "_currentElement": <App />,
      "_debugID": 23,
      "_hostContainerInfo": null,
      "_hostParent": null,
      "_instance": StatelessComponent {
        "_reactInternalInstance": [Circular],
        "context": Object {},
        "props": Object {},
        "refs": Object {},
        "state": null,
        "updater": Object {
          "enqueueCallback": [Function],
          "enqueueCallbackInternal": [Function],
          "enqueueElementInternal": [Function],
          "enqueueForceUpdate": [Function],
          "enqueueReplaceState": [Function],
          "enqueueSetState": [Function],
          "isMounted": [Function],
          "validateCallback": [Function],
        },
      },
      "_mountOrder": 1,
      "_pendingCallbacks": null,
      "_pendingElement": null,
      "_pendingForceUpdate": false,
      "_pendingReplaceState": false,
      "_pendingStateQueue": null,
      "_renderedComponent": NoopInternalComponent {
        "_currentElement": <div>
          <h2>
                    mobx-react-form-hoc demo
          </h2>
          <withForm(LoginForm) />
</div>,
        "_debugID": 24,
        "_renderedOutput": <div>
          <h2>
                    mobx-react-form-hoc demo
          </h2>
          <withForm(LoginForm) />
</div>,
      },
      "_renderedNodeType": 0,
      "_rootNodeID": 0,
      "_topLevelWrapper": null,
      "_updateBatchNumber": null,
      "_warnedAboutRefsInRender": false,
    },
    "getRenderOutput": [Function],
    "render": [Function],
  },
  "root": [Circular],
  "unrendered": <App />,
}
`;
